cmd/lakectl/cmd/dbt.go

```diff
@@ -45,53 +100,59 @@ var dbtCreateBranchSchema = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		clientType, _ := cmd.Flags().GetString("from-client-type")
 		branchName, _ := cmd.Flags().GetString("branch")
+		shouldCreateBranch, _ := cmd.Flags().GetBool(createLakefsBranchFlag)
 		toSchema, _ := cmd.Flags().GetString("to-schema")
 		projectRoot, _ := cmd.Flags().GetString("project-root")
 		skipViews, _ := cmd.Flags().GetBool("skip-views")
 		dbfsLocation, _ := cmd.Flags().GetString("dbfs-location")
 		continueOnError, _ := cmd.Flags().GetBool("continue-on-error")
 		continueOnSchemaExists, _ := cmd.Flags().GetBool(continueOnSchemaFlag)
 
+		if len(toSchema) == 0 {
+			toSchema = branchName
+		}
+
+		dbtClient := DbtClient{projectDir: projectRoot}
+		metastoreClient, clientDeferFunc := getMetastoreClient(clientType, "")
+		defer clientDeferFunc()
```

Comment:
```suggestion
		metastoreClient, clos := getMetastoreClient(clientType, "")
		defer close()
```
(name the variable for what it is, not for how it will be used on the next line).